windows.md

```diff
@@ -0,0 +1,29 @@
+Windows Node Support
+
+Windows nodes are partially supported by kubecost as of v1.93.0. Additional support is coming soon!
+* Deployment
+  * The cluster must have at least 1 linux node for the kubecost cost-model to run on
+  * Taint the cost-model deployment's pods to run on this node, eg:
+    ```
+    spec:
+      nodeSelector:
+        kubernetes.io/os: linux
+      containers:
+    ```
+  * Taint the network-costs and node-exporter daemonset to run on linux nodes only, if using network-costs and node-exporter, eg:
+    ```
+      affinity: 
+        nodeAffinity:  
+          requiredDuringSchedulingIgnoredDuringExecution:  
+            nodeSelectorTerms:  
+            - matchExpressions:  
+              - key: kubernetes.io/os 
+                operator: In 
+                values: 
+                - linux
+    ```
+
+* Metrics
+  * When tracked by the kubernetes API, nodes should show up with the correct number of pods and resources.
```

Comment:
why's this important/relevant?

Reply:
It's not clear what you're trying to say... this feels like saying "the product works when you have the Kubernetes API"

Is that what you're saying?

Reply:
Just trying to clarify expectations for what should work and what doesn't work out of the box.

Reply:
Took another pass at the language there.